Improve programmatic creation (#935)
smartcontractkit/documentation: docs/chainlink-keepers/register-upkeep.md

You can dynamically create and manage Upkeeps from within your own dApp. To do this you will need to keep track of the Upkeep ID as your contract will use this to subsequently interact with the Keepers registry. The following example displays a smart contract that can create an Upkeep and determine the Upkeep ID. Note your contract should be Keepers-compatible you will need [ERC-677 LINK](../../link-token-contracts/) to fund the Upkeep. You can also program your Upkeep to check its own balance and fund itself by interacting with the registry.

### Prerequisites 

> You need to meet these prerequisites to ensure the code snippet will work.

Reviewer: ```suggestion
You must meet the following prerequisites to ensure that this example will work:
```

> * Ensure you use the LINK, registry and registrar addresses for your network.

Reviewer 2: nit, maybe "Ensure you use the LINK token contract" instead of simply "LINK" just to be more clear?

Author: Agree, will change

> * Ensure you use the LINK, registry and registrar addresses for your network.

Reviewer: ```suggestion
- Correct [LINK token contract](../../link-token-contracts/)
- [Registry and registrar addresses for your network](#Registry-and-Registrar-Addresses) 
```

>   - LINK address per network can be found [here](../../link-token-contracts/).

Reviewer: ```suggestion
```

>   - Registry and registrar addresses are [here](#Registry-and-Registrar-Addresses) and you can also see them in the Keepers App.

Reviewer: ```suggestion
```

> * Ensure your contract has enough [ERC-677 LINK](../../link-token-contracts/) to fund the Upkeep at creation. The minimum amount is 5 LINK.

Reviewer: ```suggestion
**Note**: Make sure your contract has enough [ERC-677 LINK](../../link-token-contracts/) to fund the Upkeep at creation. _The minimum amount is 5 LINK._
```
